### PR TITLE
[PREVIEW] Mini Petition bug fix & Defended Divorce status page added

### PR DESCRIPTION
**Pull request by mounikahmcts**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #  DIV-3403 DIV-3349

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
   },
   "devDependencies": {
     "@hmcts/div-idam-test-harness": "https://github.com/hmcts/div-idam-test-harness.git",
-    "@hmcts/one-per-page-test-suite": "^2.0.11",
+    "@hmcts/one-per-page-test-suite": "https://github.com/hmcts/one-per-page-test-suite.git",
```
Comment (mounikahmcts) [inline package.json]: 2.0.13 version is not getting picked for OPPTS

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "setup": "NODE_PATH=.",
     "dev": "NODE_PATH=. node-dev server.js",
     "mocks": "NODE_PATH=. node-dev mocks/index.js",
-    "test": "NODE_PATH=. NODE_ENV=testing LOG_LEVEL=ERROR nyc mocha 'test/unit/**/*.test.js' --timeout 5000",
+    "test": "NODE_PATH=. NODE_ENV=testing LOG_LEVEL=ERROR nyc mocha 'test/unit/**/**/*.test.js' --timeout 5000",
```
Comment (mounikahmcts) [inline package.json]: test/units/view/filters/*.test.js were missed

Comment (dtaylor7) [reply package.json]: are you sure? `**/*` should include all subfolders?

File: package.json (hunk 1 above)
Comment (dtaylor7) [reply package.json]: its being picked up now, can you check?

File: steps/petition-progress-bar/PetitionProgressBar.html [hunk 3]
```diff
@@ -8,11 +8,11 @@
 
 {% block main_content %}
   <p class="font-xsmall">
-    <strong class="bold-xsmall">{{ content.referenceNumber }}:</strong>&nbsp;&nbsp;&nbsp; {{ caseId | safe }} <br>
+    <strong class="bold-xsmall">{{ content.referenceNumber }}:</strong>&nbsp;&nbsp;&nbsp; {{ caseId | space | safe }} <br>
```
Comment (dtaylor7) [inline steps/petition-progress-bar/PetitionProgressBar.html]: there is a filter called `hypenate` for the reference number... would that do the job?

File: steps/petition-progress-bar/PetitionProgressBar.html [hunk 4]
```diff
@@ -8,11 +8,11 @@
...
     <strong class="bold-xsmall">{{ content.name }}:</strong> &nbsp;&nbsp;&nbsp; {{ case.respondentFirstName }} {{ case.respondentLastName }}
   </p>
   
-  {% include "./sections/" + ccdStatus + "/PetitionProgressBar." + ccdStatus + ".template.html" %}
```
Comment (dtaylor7) [inline steps/petition-progress-bar/PetitionProgressBar.html]: 👍

File: steps/petition-progress-bar/sections/submitted/PetitionProgressBar.submitted.template.html [hunk 5]
```diff
@@ -19,7 +19,7 @@
 {% include "reform-progress-list/template.njk" %}
 
 <h2 class="heading-medium">{{ content.submittedAppStatusMsg }}</h2>
-<p>{{ content.submittedAppStatusMsgDetails1 }}<b>{{ caseId }}</b>{{ content.submittedAppStatusMsgDetails2 }}</p>
+<p>{{ content.submittedAppStatusMsgDetails1 }}<b>{{ caseId | space | safe }}</b>{{ content.submittedAppStatusMsgDetails2 }}</p>
```
Comment (dtaylor7) [inline steps/petition-progress-bar/sections/submitted/PetitionProgressBar.submitted.template.html]: again maybe use `hyphenate` filter?

**Pull request closed without merging**